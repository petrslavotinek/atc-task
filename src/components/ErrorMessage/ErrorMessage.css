.ErrorMessage {
  text-align: center;
  background-color: #FE4A49;
  color: #fff;
  padding: 20px;
  font-size: 1.1em;
  margin-top: 10px; }
