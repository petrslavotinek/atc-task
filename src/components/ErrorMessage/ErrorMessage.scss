@import "scenes/styles/_variables";

.ErrorMessage {
  text-align: center;
  background-color: $red;
  color: #fff;
  padding: 20px;
  font-size: 1.1em;
  margin-top: $spacing;
}