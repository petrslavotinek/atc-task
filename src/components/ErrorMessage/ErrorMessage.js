import React from 'react';
import PropTypes from 'prop-types';

import './ErrorMessage.css';

const propTypes = {
  message: PropTypes.string.isRequired
};

const ErrorMessage = ({
  message
}) => {
  return (
    <div className="ErrorMessage">
      {message}
    </div>
  );
};

ErrorMessage.propTypes = propTypes;

export default ErrorMessage;