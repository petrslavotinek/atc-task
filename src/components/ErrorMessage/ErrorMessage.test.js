import React from 'react';
import renderer from 'react-test-renderer';

import ErrorMessage from './ErrorMessage';

describe('ErrorMessage component', () => {
  it('should render', () => {
    expect(renderer.create(
      <ErrorMessage message="my message" />
    ).toJSON()).toMatchSnapshot();
  })
});