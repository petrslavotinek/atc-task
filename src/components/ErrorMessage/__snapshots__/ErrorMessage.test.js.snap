// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component should render 1`] = `
<div
  className="ErrorMessage"
>
  my message
</div>
`;
